camelot/view/proxy/collection_proxy.py

```diff
@@ -711,7 +711,34 @@ def gui_run(self, item_model):
         item_model.settings.setValue(self.field_name, self.width)
         item_model.settings.endGroup()
         item_model.settings.endGroup()
-        
+
+
+class ProxyRegistry:
```

Comment:
verplaats deze class aub naar de 'camelot/core/item_model' in 'proxy_registry' of zo.

het betreft hier een andere proxy dan de CollectionProxy + de hele CollectionProxy class definitie zal op termijn verschuiven naar de client, terwijl de proxy_registry server side blijft

Reply:
done